lib/helpers/index.js

```diff
@@ -47,6 +47,7 @@ function unlessEquals(a, b, options) {
 }
 
 function slugify(value) {
+    value = value || '';
```

Comment:
is this about a default value for `value`, or about not treating other falsy values for `value` as a string?

`function slugify(url = '') {`

or `if (!value) return ''`

Reply:
It was about not treating other falsy values as strings. We lint against `if (!value) return ''`, but I could not be lazy and write it more verbosely...